### PR TITLE
Add note to OpenSource.md about squashing history. Minor spelling, etc.

### PR DESCRIPTION
**Pull request by ninjabem**
Please excuse the OCD.

### CODE REVIEW

**Comment by paololim** [comment]
:) Lgtm

**Comment by mbryzek** [comment]
lgtm

File: Lifecycle.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ We pursue several practices to ensure our SDLC achieves its goals of providing v
   
     - isolation - truly ensuring that one change cannot impact another. For example, our focus on API design and micro service architecture on the backend is one technique we use to create isolation.
     
-    - continuous delivery - by releasing small changes continuously to our production environment, we minimze the impact of any one change and significantly increase time to resolution.
+    - continuous delivery - by releasing small changes continuously to our production environment, we minimze the impact of any one change and significantly decrease time to resolution.
```
Comment (ericluria) [inline Lifecycle.md]: Missed one here! "Minimize"

File: OpenSource.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ public.
   - Discuss your project with 1-2 team members - it's helpful to talk to both people familiar with your work and also people from other teams.
   - Update your documentation - and specifically your README - to be as clear as possible. Keep in mind that people will be learning about your project with no background. We recommend taking a look at the documentation for other open source projects you love as a template.
   - Clean up your config - specifically want to confirm there are no specifics of Flow's environment
-  - Review the project and its history to make sure there are no usernames, passwords, credetials, etc. If necessary, create a new repo.
+  - Review the project to make sure there are no usernames, passwords, credetials, etc. If necessary, create a new repo.
```
Comment (ericluria) [inline OpenSource.md]: Credentials

File: Lifecycle.md (hunk 1 above)
Comment (ninjabem) [inline Lifecycle.md]: nice!

File: OpenSource.md (hunk 2 above)
Comment (ninjabem) [inline OpenSource.md]: thanks!

**Pull request merged**